src/pages/program/Program.tsx

```diff
@@ -37,12 +39,13 @@ const Program: React.FC<ProgramProps> = () => {
   return (
     <ContentWrapper>
       <PageTitle>Program</PageTitle>
-      <Accordion>
+      <Accordion defaultActiveKey="1">
         <Card>
           <Accordion.Toggle as={Card.Header} eventKey="0">
             <TimeplanEntry>
               <Time>13:15 - 14:15</Time>
               Lynpresentasjoner i store auditorium
+              <Icon path={mdiChevronDown} size="2em" />
```

Comment:
Jeg ville lagt in en `onClick` listener som endrer ikonet når de er åpnet/lukket, det er litt klussete når det er samme ikon for en åpen og en lukket accordion.

Reply:
Ja, tenkte litt på det! Burde ikonet fjernes eller bare bli mindre tydeligere eller?

Reply:
Enig med @sondr3. Ikonet burde bare roteres, tror eg 😊 F.eks noe ala https://codepen.io/zabielski/pen/ZGLbav

Reply:
De har vel også et ikon som heter `mdiChevronUp` som kan brukes? Da kan du bare ha en `onClick` som endrer ikonet :robot: